math/src/field/fields/fft_friendly/babybear.rs

```diff
@@ -0,0 +1,102 @@
+use crate::{
+    field::{
+        element::FieldElement,
+        fields::montgomery_backed_prime_fields::{IsModulus, MontgomeryBackendPrimeField},
+        //traits::IsFFTField,
+    },
+    unsigned_integer::element::UnsignedInteger,
+};
+
+pub type U64 = UnsignedInteger<1>;
+pub type U64PrimeField<T> = MontgomeryBackendPrimeField<T, 1>;
```

Comment:
There's already a `U64PrimeField` ([here](https://github.com/lambdaclass/lambdaworks/blob/29af640a8e4e22301e7866981f156281d03ab51a/math/src/field/fields/u64_prime_field.rs#L15)). 
@MauroToscano @Oppen I guess there's no need to have both u64primefields if the one with the Montgomery backend is more performant. Maybe we should create an issue to benchmark them and remove the slow one.
For the time being I would change the name of this type alias to `U64MontgomeryBackendPrimeField` or something along those lines. To avoid confusion.

Reply:
We should review this. But let's do it ourselves, and discuss it outside this PR.

Reply:
The change of name would be nice though